monailabel/main.py

```diff
@@ -325,9 +325,15 @@ def start_server_init_settings(self, args):
             logger.debug("                  ENV VARIABLES/SETTINGS                  ")
             logger.debug("**********************************************************")
             for k, v in settings.dict().items():
-                v = json.dumps(v) if isinstance(v, list) or isinstance(v, dict) else str(v)
+                if isinstance(v, list) or isinstance(v, dict):
+                    v = json.dumps(v)
+                elif v is not None:
+                    v = str(v)
+                else:
+                    v = None
```

Comment:
```suggestion
```
Could be ommitted.

Reply:
I agree, feel free to delete the else block